### ISSUE TITLE
ROS2 support 

### ISSUE THREAD

**Issue by ajay1606** [LTU-RAI/Dsp#5]
@grammers Any plans for ROS2 support !
Had any chance to test in ROS2 ?

**Comment by grammers** [comment]
Yes branch ros2_msgs. The name is misleading but it is the full pkg.
It is working in the same way as for ros1. However, the support scrip (path to point) is not ported.
